Gemfile.lock

```diff
@@ -257,6 +262,7 @@ GEM
     zeitwerk (2.6.12)
 
 PLATFORMS
+  x64-mingw-ucrt
   x86_64-linux
 
 DEPENDENCIES
```

Comment:
**README.md** file:

- [ ] To make your README looks more professional and as required the used template, kindly ensure that you update the `## Authors` section of your README by adding all authors contributors of the project